### ISSUE TITLE
Change the codebase 

### ISSUE THREAD

**Issue by heysagnik** [heysagnik/screenREC#72]
### Convert the codebase from Pug to Next.js

**Comment by taimooralqureshi** [comment]
it could cause regression. don't seem a `good first issue`. what is that bug in that?

**Comment by heysagnik** [comment]
> it could cause regression. don't seem a `good first issue`. what is that bug in that?

No bug but not everyone familiar with pug so thought to convert it to Next Js

**Comment by LoganTann** [comment]
The code is very difficult to work with, since there are no components and the code style is a bit bad. I'd be happy to learn NextJs and try moving the whole codebase into this framework.

What do you think @heysagnik ?

**Comment by LoganTann** [comment]
However, I'm much more confident with Vue.JS // Nuxt

If you agree, I prefer working with this framework instead of a react-based one.

**Comment by heysagnik** [comment]
> However, I'm much more confident with Vue.JS // Nuxt
> 
> If you agree, I prefer working with this framework instead of a react-based one.

Are you on Discord, would love to talk with you regarding this?

**Comment by heysagnik** [comment]
> However, I'm much more confident with Vue.JS // Nuxt
> 
> If you agree, I prefer working with this framework instead of a react-based one.

@LoganTann can you work on this 😅?

**Comment by LoganTann** [comment]
yes, but please create a new repo since there will be several PRs with breaking changes

**Comment by LoganTann** [comment]
EDIT : no need to create a repo, just create a new branch, please.

**Comment by heysagnik** [comment]
> EDIT : no need to create a repo, just create a new branch, please.

Okay you create a PR first

**Comment by adnanxali** [comment]
Is this issue assigned to anybody for hacktober fest 2023?. If not can I try to work on this.

**Comment by heysagnik** [comment]
@adnanxali you may work if you can, already @LoganTann worked to port to vue js #88 
If you are good in vue contribute in dev branch or you may continue in any language

**Comment by adnanxali** [comment]
okay thanks i'll try.

**Comment by sahilatahar** [comment]
Hey there!

I wanted to share with you that I've built a new version of this project using modern technologies like Next.js, React.js, and Tailwind CSS. The new version includes additional features and an improved UI. I'd love for you to check it out and provide any feedback or contributions.

You can find the new repository here: https://github.com/sahilatahar/MediaMaster

Looking forward to hearing your thoughts!
